src/app/(frontend)/[center]/observations/page.tsx

```diff
@@ -46,9 +47,26 @@ export default async function Page({ params }: Args) {
 
   const { version, baseUrl } = await getNACWidgetsConfig()
 
+  // Build query params, preserving array values
+  const queryParams = new URLSearchParams()
```

Comment:
Decided to keep this here because it would be more work to move to `WidgetHashHandler`. We would need to keep track of `initialHash` (which differ a lot) & `cleanUrl` which we could easily do with `pathname`.. but still not worth it.